### PR TITLE
Smaller updates to staking documentation, adjust operator node commands

### PR DESCRIPTION
**Pull request by EmilFattakhov**
As per testing, there was a missing step at starting an operator node, and some of the required flags weren't set, while some were extra.

### CODE REVIEW

File: docs/operators_and_staking/operators.md [hunk 1]
```diff
@@ -185,11 +192,13 @@ target/production/subspace-node `
 
 ```bash
 target/production/subspace-node \
---chain gemini-3g \
+--chain gemini-3g
```
Comment (jim-counter) [inline docs/operators_and_staking/operators.md:195]:
```suggestion
--chain gemini-3g \
```

File: docs/operators_and_staking/operators.md [hunk 2]
```diff
@@ -202,11 +211,13 @@ target/production/subspace-node \
...
```
Comment (jim-counter) [inline docs/operators_and_staking/operators.md:214]:
```suggestion
--chain gemini-3g \
```

File: docs/operators_and_staking/operators.md [hunk 3]
```diff
@@ -216,11 +227,7 @@ target/production/subspace-node \
 </TabItem>
 </Tabs>
 
-:::note
-Make sure to use the public key (hex) that we generated earlier in the [Create Operator Key](#create-operator-key) section
-:::
-
-You should see the node start sucesfully and begin syncing
+You should see the node start sucesfully and begin syncing.
```
Comment (jim-counter) [inline docs/operators_and_staking/operators.md:230]:
```suggestion
You should see the node start successfully and begin syncing. 
```

File: docs/operators_and_staking/operators.md [hunk 4]
```diff
@@ -320,5 +327,4 @@ Only the account who registered **Operator** can swith the domain.
 
 :::note
 Stake of your **Nominators** won't be released, but will be moved to the new domain as well. 
-:::
-
+:::
```
Comment (jim-counter) [inline docs/operators_and_staking/operators.md:330]: Was adding this intended?

File: versioned_docs/version-latest/operators_and_staking/operators.md [hunk 5]
```diff
@@ -185,11 +192,13 @@ target/production/subspace-node `
 
 ```bash
 target/production/subspace-node \
---chain gemini-3g \
+--chain gemini-3g
```
Comment (jim-counter) [inline versioned_docs/version-latest/operators_and_staking/operators.md:195]:
```suggestion
--chain gemini-3g \
```

File: versioned_docs/version-latest/operators_and_staking/operators.md [hunk 6]
```diff
@@ -185,11 +192,13 @@ target/production/subspace-node `
...
 --rpc-external \
---node-key 0xxeea96fe9cfbd6c1d7e9657e36447a158c0c80432b2bc8869a1c6706707843f \
--- \
---domain-id 3 \
+--bootnodes /ip4/3.87.28.170/tcp/40333/p2p/12D3KooWGHtULvhdKMZtzigSK1438uWXPj9rBQHVzTaKMWv1WRXp \
+--name your_node_name \
+--base-path your_path_to_node_data \
+-- `
```
Comment (jim-counter) [inline versioned_docs/version-latest/operators_and_staking/operators.md:200]:
```suggestion
-- \
```

File: docs/operators_and_staking/operators.md [hunk 7]
```diff
@@ -202,11 +211,13 @@ target/production/subspace-node \
...
---chain gemini-3g \
+--chain gemini-3g
 --rpc-external \
---node-key 0xxeea96fe9cfbd6c1d7e9657e36447a158c0c80432b2bc8869a1c6706707843f \
--- \
---domain-id 3 \
+--bootnodes /ip4/3.87.28.170/tcp/40333/p2p/12D3KooWGHtULvhdKMZtzigSK1438uWXPj9rBQHVzTaKMWv1WRXp \
+--name your_node_name \
+--base-path your_path_to_node_data \
+-- `
```
Comment (jim-counter) [inline docs/operators_and_staking/operators.md:219]:
```suggestion
-- \
```

File: docs/operators_and_staking/operators.md [hunk 8]
```diff
@@ -185,11 +192,13 @@ target/production/subspace-node `
...
```
Comment (jim-counter) [inline docs/operators_and_staking/operators.md:200]:
```suggestion
-- \
```

File: versioned_docs/version-latest/operators_and_staking/operators.md [hunk 9]
```diff
@@ -202,11 +211,13 @@ target/production/subspace-node \
...
```
Comment (jim-counter) [inline versioned_docs/version-latest/operators_and_staking/operators.md:219]:
```suggestion
-- \
```

File: versioned_docs/version-latest/operators_and_staking/operators.md [hunk 10]
```diff
@@ -216,10 +227,6 @@ target/production/subspace-node \
 </TabItem>
 </Tabs>
 
-:::note
-Make sure to use the public key (hex) that we generated earlier in the [Create Operator Key](#create-operator-key) section
-:::
-
 You should see the node start sucesfully and begin syncing
```
Comment (jim-counter) [inline versioned_docs/version-latest/operators_and_staking/operators.md:230]:
```suggestion
You should see the node start successfully and begin syncing
```

**Review by jim-counter — CHANGES_REQUESTED** [review]
Thanks Emil, a few small typos found. I've updated the doc we're working from with what's been covered. Will we not remove `--rpc-external` too?

File: docs/operators_and_staking/operators.md (hunk 4 above)
Comment (EmilFattakhov) [reply docs/operators_and_staking/operators.md:330]: It's just a formatting highlight, I suppose. No real changes here

**Review by jim-counter — APPROVED** [review]

File: versioned_docs/version-latest/operators_and_staking/operators.md [hunk 11]
```diff
@@ -322,3 +323,77 @@ Only the account who registered **Operator** can swith the domain.
...
+--no-private-ipv4 \
+--validator \
+--name your_node_name \
+# operator flags
+-- \
+--domain-id 1 \
+--operator \
+--keystore-path /tmp/keystore \
+--bootnodes /ip4/3.87.28.170/tcp/40333/p2p/12D3KooWGHtULvhdKMZtzigSK1438uWXPj9rBQHVzTaKMWv1WRXp
+--bootnodes /ip4/3.87.28.170/tcp/40333/p2p/12D3KooWGHtULvhdKMZtzigSK1438uWXPj9rBQHVzTaKMWv1WRXp `
```
Comment (jim-counter) [inline versioned_docs/version-latest/operators_and_staking/operators.md:374]:
```suggestion
```

**Review by jim-counter — CHANGES_REQUESTED** [review]
Could we move the registration so it is after the sync. With a note warning not to register until in sync please?

File: versioned_docs/version-latest/operators_and_staking/operators.md (hunk 11 above)
Comment (EmilFattakhov) [reply versioned_docs/version-latest/operators_and_staking/operators.md:374]: Oops, didn't push the last commit which fixed this, my bad.

**Pull request merged**